nengo/ipynb.py

```diff
@@ -45,6 +45,12 @@
 
 
 def load_ipython_extension(ipython):
+    # Not a deprecation warning as this are hidden by default and we want to
+    # make sure the user sees this message.
+    warnings.warn(
+        "Loading the nengo.ipynb notebook extension will break with current "
+        "Jupyter notebook versions and is deprecated. All features provided "
+        "by the old extension are automatically activated for IPython>=5.")
```

Comment:
Why are we keeping this extension around? For IPython versions 2-4? At the very least, we should ensure that this becomes a no-op if using IPython >= 5... but I would also be okay making it a no-op for all versions. IPython versions 2-4 having a terminal progress bar seems fine to me. If anyone really wants it back then can go back to previous Nengo versions.

Reply:
* There will be a number of existing notebooks that still load the extension. It seems like a bad idea to break them.
* No-op is fine with me if we completely remove the old IPython progress bar.
* Using the `TerminalProgressBar` without a special `ProgressUpdater` in IPython 2 can consume a lot of memory and might increase the notebook size to be large enough to crash your browser. (This is fixed in IPython 3.)